supplementary/alica_ros_turtlesim/ros2/CMakeLists.txt

```diff
@@ -53,6 +59,9 @@ target_link_libraries(${PROJECT_NAME}
     ${geometry_msgs_LIBRARIES}
 )
 
+add_subdirectory(../libalica-turtlesim ../libalica-turtlesim)
```

Comment:
same

Reply:
Because the "subdirectory" is not an actual subdirectory of ros1 we have to provide a relative build path for it. This allows us to build alica_ros_turtlesim without it being a package